__tests__/app.test.js

```diff
@@ -34,4 +62,25 @@ describe("app",()=>{
             })
         })
     })
+    describe("GET /api",()=>{
+        test("Status Code: 200 and should return api endpoints", () => {
+            return request(app)
+            .get("/api/")
+            .expect(200)
+            .then(({body})=>{
+                expect(endpointFormat).toMatchObject(body.apiEnd)
```

Comment:
this is quite a brittle test, when you update the endpoints.json for future tickets this test will fail unless you update the object you made here. A better way to do it is just to require in the endpoints.json file, then compare that against your body.apiEnd